spec/features/create_answer_spec.rb

```diff
@@ -22,6 +22,14 @@
     end
   end
 
+  scenario 'User tries to create invalid answer', js: true do
+    sign_in user
+    visit question_path(question)
+    click_on 'Submit'
+
+    expect(page).to have_content "Body is too short"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.